magento/pwa-studio
# [Doc] peregrine targets ref docs

## Description

This PR updates the JSDocs in the target intercept and declare files in the Peregrine package.
It also creates a file in devdocs that pulls this information into a reference-style topic.

## Related Issue

Closes PWA-681

## Acceptance 

@zetlen 

### Verification Stakeholders

@zetlen 

### Specification



### Verification Steps
1. Navigate to the pwa devdocs directory: `cd pwa-devdocs`
1. Start the HTML preview server: `yarn develop`
1. Verify files are auto-generated:

    - `src/_includes/auto-generated/peregrine/lib/targets/peregrine-intercept.md`
    - `src/_includes/auto-generated/peregrine/lib/targets/peregrine-declare.md`

1. On the devdocs preview site, in the top navigation, click on **Peregrine API** under the _Reference API_ item.
1. Verify the Extensibility Targets topic appears with content.
1. Verify this topic is listed on the left navigation for this section
1. Verify the Wrappable Talons topic is listed on the left navigation and links to the correct page 

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
